protractor/e2e/integration/ng2.spec.ts

```diff
@@ -58,5 +58,13 @@ describe('Main page test suite', () => {
         await secondPage.infoSubmitButton.click();
     });
 
+    it('Check blog page',async () => {
+        await secondPage.blogButton.click();
+        expect(await secondPage.getCurrentUrl()).toContain(secondPage.blogLink);
+        expect(await secondPage.postsList.count()).toEqual(20);
+        expect(secondPage.isElementDisplayed(await secondPage.navigationBar)).toBeTruthy();
```

Comment:
And user see navigation on the bottom: "First Page 1 2  NextLast Page" - this is task
You check only visibility of navigation bar, so need to add other checks, like:
expect(await $('.paging-first').isDisplayed()).toBeTruthy();
        expect(await $('.paging-last').isDisplayed()).toBeTruthy();
        expect(await $('.paging-next').isDisplayed()).toBeTruthy();
        expect(await $$('li .page-numbers').count()).toBe(2);